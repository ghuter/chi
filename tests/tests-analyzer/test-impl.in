Stringer :: signature {
	string :: fun () ^u8;
}

HelloString :: impl Stringer {
	string :: fun () ^u8 {
		return "hello";
	}
}
