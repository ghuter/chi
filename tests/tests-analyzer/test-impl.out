SMODSIGN(Stringer, [], (), {SSIGN(string) : ^u8})
SMODIMPL(HelloString, [], (), | |, {SFUN(string, {SRETURN(ECSTS(hello))}) : ^u8}) : Stringer
